seml/main.py

```diff
@@ -164,6 +168,15 @@ def main():
 
     args = parser.parse_args()
 
+    if args.configure:  # launch SEML configure.
+        args.func = mongodb_credentials_prompt
+        del args.config_file
+        del args.configure
+    else:  # otherwise remove the flag as it is not used elsewhere.
+        del args.configure
+        if args.config_file is None or len(args.config_file) == 0:
```

Comment:
If you use `default=None` you don't have to check the length here.

Reply:
Ok, see 4b573e0.